.github/workflows/ccpp.yml

```diff
@@ -67,7 +67,7 @@ jobs:
     - name: Run Tests
       run: |
            cd gccrs-build; \
-           make check-rust
+           make check-rust RUNTESTFLAGS="--target_board=unix\{-m32,-m64}"
```

Comment:
I think it would be worth it to also do it on gcc 4.8 as clearly this is a surprising endeavor haha. I don't know if that causes issues on MacOS so I don't know if it's worth adding. @simonpcook what do you think?

I think it's fine to just add that `--target_board=unix\{-m32, -m64}` to the gcc 4.8 step, but eventually if we keep adding these options it'd be nice to make use of the github action matrices to run these jobs in parallel. Just a note for future modifications.